plover_casecat_dictionary.py

```diff
@@ -49,38 +49,38 @@ def _parse(self, filename):
             while True:
 
                 unknown1 = f.read(18)
-                if unknown1 is None or len(unknown1)!=18:
+                if unknown1 is None or len(unknown1) != 18:
                     return
 
                 stroke_count = unpack(">B", f.read(1))[0]
                 letter_count = unpack(">B", f.read(1))[0]
 
-                unknown2 = f.read(1)
+                f.read(1)
 
-                stroke = []
-
-                for i in range(stroke_count):
-                    stroke.append('')
+                strokes = []
 
+                for _ in range(stroke_count):
+                    stroke = ''
                     stroke_bits = unpack(">L", f.read(4))[0]
 
-                    for k in range(len(KEYS)):
-
-                        bit = (0x80000000) >> k
-                        if KEYS[k]=='*' and stroke[-1]=='':
-                            stroke[-1] += '-'
+                    for k, steno_key in enumerate(KEYS):
+                        bit = 0x80000000 >> k
+                        if steno_key == '*' and stroke == '':
```

Comment:
I think this should be if steno_key == 'F'

Reply:
you're right!